core/src/main/resources/error/error-classes.json

```diff
@@ -2101,6 +2101,11 @@
     ],
     "sqlState" : "428EK"
   },
+  "THRESHOLD_VALUE_OUT_OF_RANGE" : {
```

Comment:
Add an error class, or should we keep to using `DATATYPE_MISMATCH.VALUE_OUT_OF_RANGE`? @cloud-fan